hooli_data_eng/resources/snowflake_io_manager.py

```diff
@@ -88,29 +89,30 @@ class SnowflakeIOManager(IOManager):
     """
 
     def get_output_asset_key(self, context: OutputContext):
-        return generate_asset_key_for_snowflake_table(context.metadata["table"])
+        if "assets" not in context.pipeline_name:
```

Comment:
is there a key that should be returned if `asset` is in `context.pipeline.name`?